clients/venice-thin-client/src/main/java/com/linkedin/venice/client/schema/RouterBasedStoreSchemaFetcher.java

```diff
@@ -74,6 +74,33 @@ public Schema getLatestValueSchema() {
     }
   }
 
+  @Override
+  public Schema getSupersetSchema() {
+    String valueSchemaRequestPath = TYPE_VALUE_SCHEMA + "/" + storeClient.getStoreName();
+    MultiSchemaResponse multiSchemaResponse = fetchAllValueSchemas(valueSchemaRequestPath);
```

Comment:
We have an endpoint for fetching the superset schema specifically... why do we need to fetch all schemas? Is it that the endpoint is only on the controller and we want to query the router instead?

Reply:
Brain fart 🤦 . Will update to use the API directly. There is no use of fetching all schemas

Reply:
It doesn't look like we had an API for fetching superset schema, but I added one for routers as that seems the better way to handle this